.github/workflows/crossterm_test.yml

```diff
@@ -47,9 +47,24 @@ jobs:
     - name: Test Build
       run: cargo build
       continue-on-error: ${{ matrix.can-fail }}
-    - name: Run Tests
+    - name: Test all features
       run: cargo test --all-features -- --nocapture --test-threads 1
```

Comment:
Why do we test all features separately after testing all of them here?

Reply:
Terminal auto enables Cursor feature for example. With all features test, we enable everything and test it. But if anyone makes a mistake in Cargo.toml, terminal wont auto enable cursor, it will still pass for all features, but will fail if user wants a terminal only. In other words, we have to test all features and one by one to catch these kind of errors.